tests/test_partisn.py

```diff
@@ -216,14 +215,7 @@ def test_get_coord_sys_3D():
 def get_zones_no_void():
     """Test the _get_zones function if no void is in the meshed area."""
 
-    try:
-        from pyne import dagmc
-    except:
-        raise SkipTest
-
-    if not HAVE_PYMOAB:
-        raise SkipTest
-
+    from pyne import dagmc
```

Comment:
don't we need to do the same set of exception handling and tests for `HAVE_PYMOAB` here?